src/app/components/Providers/LastfmProvider.tsx

```diff
@@ -162,18 +193,69 @@ const LastfmProvider: React.FC<LastfmProviderProps> = ({ children, mk }: LastfmP
     fetchToken();
   }, []);
 
-  useEffect(() => {
-    if (mk.mediaItem && mk.mediaItem.item) {
-      scrobble(mk.mediaItem.item);
-    }
-  }, [mk.mediaItem]);
+  useMK({
```

Comment:
You are not using the return value of useMK, choose one of useMK or withMK

Reply:
Thanks, I had missed the `withMK` before.
I removed `useMK` and added an event listener to the `withMK` bindings instead.